build/index.html

```diff
@@ -52,8 +52,7 @@
 	<meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,minimal-ui,user-scalable=0">
 	<meta name="e840127dba35b2939858e9ed30d25cb70b353e25" content="117ae21d29f8732b0bde7381da50aa67b1548bd9" />
 
-	<style type="text/css">#app,#splash,body,html{width:100%;height:100%}#splash,#splash-text{top:0;left:0;overflow:hidden}#splash{display:block;position:relative;color:#f5f5f5;background:#efefef}#splash-text{display:inline-block;position:absolute;bottom:0;right:0;width:50px;height:50px;margin:auto;color:#fbfbfb;background-image:url(2.0.0/assets/images/augur_logo_200_white.png);background-repeat:no-repeat;background-position:center center;background-size:contain;font-family:Impact,"Arial Black","Avant Garde",Charcoal,sans-serif;font-size:20vh;font-weight:700;text-shadow:0 0 50px #e9e9e9;text-align:center;animation-name:splash-anim;animation-duration:10s;animation-iteration-count:infinite;animation-timing-function:cubic-bezier(.11,.63,.25,.89);animation-direction:alternate}@keyframes splash-anim{0%{transform:scale(1) rotate(0)}100%{transform:scale(4) rotate(3deg)}}
-</style>
+	<style type="text/css">#app,#splash,body,html{width:100%;height:100%}#splash,#splash-text{top:0;left:0;overflow:hidden}#splash{display:block;position:relative;color:#f5f5f5;background:#efefef}#splash-text{display:inline-block;position:absolute;bottom:0;right:0;width:50px;height:50px;margin:auto;color:#fbfbfb;background-image:url(2.0.0/assets/images/augur_logo_200_white.png);background-repeat:no-repeat;background-position:center center;background-size:contain;font-family:Impact,"Arial Black","Avant Garde",Charcoal,sans-serif;font-size:20vh;font-weight:700;text-shadow:0 0 50px #e9e9e9;text-align:center;animation-name:splash-anim;animation-duration:10s;animation-iteration-count:infinite;animation-timing-function:cubic-bezier(.11,.63,.25,.89);animation-direction:alternate}@keyframes splash-anim{0%{transform:scale(1) rotate(0)}100%{transform:scale(4) rotate(3deg)}}</style>
```

Comment:
Inline CSS is something we're trying to stay away from.  Styling should all go in AURC.  (Even after the repos are merged, we still don't want inline CSS.)